_rules/image-button-accessible-name-59796f.md

```diff
@@ -55,128 +55,105 @@ There is a known combination of a popular browser and assistive technology that
 
 #### Passed Example 1
 
-Image button element with [accessible name][] through `alt` attribute
+The image button has an [accessible name][] through the `alt` attribute.
 
 ```html
-<input type="image" name="submit" src="button.gif" alt="Submit" />
+<input type="image" src="/test-assets/shared/search-icon.png" alt="Search" />
 ```
 
 #### Passed Example 2
 
-Image button element with [accessible name][] through `aria-label`
+The image button has an [accessible name][] through the `aria-label` attribute.
 
 ```html
-<input type="image" name="submit" src="button.gif" aria-label="Submit" />
+<input type="image" src="/test-assets/shared/search-icon.png" aria-label="Search" />
 ```
 
 #### Passed Example 3
 
-Image button element with [accessible name][] through `title` attribute
+The image button has an [accessible name][] through the `title` attribute.
 
 ```html
-<input type="image" name="submit" src="button.gif" title="Submit" />
+<input type="image" src="/test-assets/shared/search-icon.png" title="Search" />
 ```
 
 #### Passed Example 4
 
-Image button element with [accessible name][] through `aria-labelledby`
+The image button has an [accessible name][] through the `aria-labelledby` attribute.
 
 ```html
-<input type="image" name="submit" src="button.gif" aria-labelledby="id1" />
-<div id="id1">Submit</div>
-```
-
-#### Passed Example 5
-
-[accessible name][] is not empty.
-
-```html
-<input type="image" name="submit" src="button.gif" alt=":-)" />
-```
-
-#### Passed Example 6
-
-Image button element with [accessible name][] through `alt` attribute
-
-```html
-<input type="image" name="submit" src="button.gif" alt="123" />
+<input type="image" src="/test-assets/shared/search-icon.png" aria-labelledby="id1" />
+<div id="id1">Search</div>
 ```
 
 ### Failed
 
 #### Failed Example 1
 
-Image button element with empty [accessible name][].
+The image button element has an empty [accessible name][]. The `name` attribute can not be used to provide an [accessible name][].
 
 ```html
-<input type="image" name="submit" src="button.gif" />
+<input type="image" name="search" src="/test-assets/shared/search-icon.png" />
 ```
 
 #### Failed Example 2
 
-Image button element with empty [accessible name][].
+The image button has an empty `alt` attribute, and no other attributes that can give it an [accessible name][].
 
 ```html
-<input type="image" name="submit" src="button.gif" alt="" />
+<input type="image" src="/test-assets/shared/search-icon.png" alt="" />
```

Comment:
As in fail example 1, the usual methods for finding an accessible name for this element fail so it should get an accessible name of "Submit Query"

Reply:
This will still fail, because of the empty alt the default accessible name isn't used.